test/src/sdk-harness/test_artifacts/storage_vec/svec_u8/src/main.sw

```diff
@@ -0,0 +1,73 @@
+contract;
+
+use std::option::*;
+use std::result::*;
+use std::storage::{StorageVec, StorageVecError};
+
+abi MyContract {
+    #[storage(read, write)]
+    fn vec_u8_push(value: u8);
```

Comment:
Well its in a folder called StorageVec, so should be clear that svec is an abbreviation for storagevec, im using the abbreviation since theres gonna be like 10 projects all with similar names in that folder

I removed vec_ prefix for the functions, however ill keep the u8_ as it'll help in differentiating while testing